connector/src/main/resources/config.yml

```diff
@@ -130,4 +136,4 @@ enable-proxy-connections: false
 # 1400 is the default.
 # mtu: 1400
 
-config-version: 4
+config-version: 5
```

Comment:
Historically I haven't bumped the config version unless a config change is made that breaks old configs.

Reply:
Sure, will revert it